### DISCUSSION TITLE
strings referencing other strings

### DISCUSSION THREAD

**Discussion by endquote** [WeblateOrg/weblate#9426] [General]
I'm exploring a migration from an existing tool to Weblate.

In the existing tool, a string can reference another like this:

string key `appname` = "My Great App"
string key `gotoapp` = "Go to [$$appname$$]"

Then if I go look at `gotoapp` it reads as "Go to My Great App".

Does this concept exist in Weblate? Apologies if I'm just missing it in the documentation.

**Comment by nijel** [comment]
Such variables are typically expanded when rendering the translations, not when doing them. Weblate has support of checking many format strings, see https://docs.weblate.org/en/latest/user/checks.html#formatted-strings
